common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5573,6 +5575,14 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(true)
 
+  val METRICS_PUSH_APPLICATION_ID: ConfigEntry[Boolean] =
+    buildConf("celeborn.metrics.pushApplicationId")
```

Comment:
how about `celeborn.metrics.applicationIdAsLabel` or `celeborn.metrics.label.applicationId`?

Reply:
`celeborn.metrics.label.applicationId` sgtm.

Please also clarify the side effects of enabling this feature in the `docs` - application id is considered as a high cardinality label, be careful enabling it on metrics systems that are not optimized for high cardinality values.

Reply:
Currently, we rely on the application dimension metrics to get the top applications.

```
topk(50, sum by (applicationId) (metrics_diskBytesWritten_Value{role="worker", applicationId!=""}))
```

https://github.com/apache/celeborn/pull/2949

Reply:
but the top 50 application ids are changed over time, right? it also should be considered as a high cardinality label, for metrics systems like Prometheus, storing high cardinality labels is a disaster ...

Reply:
done, changed the config name. I've also made the config false by default.

>for metrics systems like Prometheus, storing high cardinality labels is a disaster
We are facing this exact issue.